createmsi.py

```diff
@@ -250,6 +251,15 @@ def generate_files(self):
             for r in self.registry_entries:
                 self.create_registry_entries(registry_entries_component, r)
 
+        if self.custom_actions is not None:
+            install_execute_sequence = ET.SubElement(package, 'InstallExecuteSequence')
+            ET.SubElement(package, 'Property', {
+                'Id': 'cmd',
+                'Value': 'cmd.exe'
```

Comment:
This hardcodes the fact that `cmd.exe` is used. Is it strictly necessary and does it work in all cases (i.e. when the thing to run is a GUI app).